### ISSUE TITLE
Error trying sending messages

### ISSUE THREAD

**Issue by ajmasia** [GokulDas027/TelegramBridge#6]
Hi guys,

When I try to send a message I get this error:
> 2020/10/03 16:49:57 unable to send message: Bad Request: can't parse entities: Character '.' is reserved and must be escaped with the preceding '\'

This is my job config:
```yml
notifier:
    if: always()

    needs:
      - linter
      - tests
    runs-on: ubuntu-latest

    steps:
      - name: Telegram notifier
        uses: GokulDas027/TelegramBridge@master
        with:
          chat: ${{ secrets.TELEGRAM_CHAT_ID }}
          token: ${{ secrets.TELEGRAM_TOKEN }}
          status: ${{ job.status }}
          event: ${{ github.event_name }}
          actor: ${{ github.actor }}
```

What could be happening? 
I have tested the bot with other actions and it works perfectly. 

Thanks!

**Comment by GokulDas027** [comment]
Hi, 

It could be due to the parsing error of "." by the telegram parser.
let me check

**Comment by GokulDas027** [comment]
@ajmasia , The issue should be fixed in v1.2 just released.
Can you please check on your repo

**Comment by ajmasia** [comment]
> @ajmasia , The issue should be fixed in v1.2 just released.
> Can you please check on your repo

Now works fine! 
Thanks for your fast answer 😄
